gateway/api/models.py

```diff
@@ -32,11 +32,9 @@ class JobConfig(models.Model):
         null=True,
     )
 
-    PYTHON_V3_8 = "py38"
```

Comment:
Yep, I totally agree with you, @psschwei. If we are not going to migrate it my proposal is more or less what you did here:
- Modify the serializer
- I would leave this model without changes

So the serializer is what we use to manage valid python versions. Other proposals @IceKhan13 , @akihikokuroda ?

Reply:
I'm good with that :)